src/Hotkeys.ts

```diff
@@ -176,9 +212,13 @@ const makeSlot = (key: string, descr: string) => {
     });
 
     const p = document.createElement('p');
-    p.id = 'hotKeyDesc';
     p.textContent = descr;
 
+    p.addEventListener('click', () => {
+        DOMCacheGetOrSet('hotkeyDescription').textContent = hotkeys.get(key)?.[3] ?? '';
+        hotkeys.get(key)![1]();
```

Comment:
this logic doesn't make sense on one line you use optional chaining and the next you assert the value cannot be null/undefined

Reply:
Didn't need it as pointed out